src/bin/pg_autoctl/cli_do_misc.c

```diff
@@ -491,33 +491,28 @@ keeper_cli_promote_standby(int argc, char **argv)
 
 
 /*
- * keeper_cli_receiwal allows to run pg_receivewal targetting the current
- * primary conninfo of a standby node, either to check that the connection
- * makes sense, or to actually start a sub-process receiving WAL.
+ * keeper_cli_identify_system connects to a Postgres server using the
```

Comment:
Minor: it might be good to add a sentence about what is `IDENTFY_SYSTEM`

Reply:
Added!